index.html

```diff
@@ -237,16 +237,16 @@ <h1>Let's chat</h1>
                 <div class="chat-desc">
                     <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
                 </div>
-                <form action="https://formspree.io/f/xbjbrdyg" method="post" id="form">
+                <form action="https://formspree.io/f/xbjbrdyg" method="post" id="form" onblur="saveInput();">
                     <div class="flname">
-                        <input type="text"  id="nname" name="nname" class="nname" placeholder="Name" maxlength="30">
-                        <input type="text"  id="fname" name="fname" class="fname" placeholder="First Name" maxlength="15">
-                        <input type="text"  id="lname" name="lname" class="lname" placeholder="Last Name" maxlength="15">
+                        <input type="text"  id="nname" name="nname" class="nname" onblur="saveInput();" placeholder="Name" maxlength="30">
+                        <input type="text"  id="fname" name="fname" class="fname" onblur="saveInput();" placeholder="First Name" maxlength="15">
+                        <input type="text"  id="lname" name="lname" class="lname" onblur="saveInput();" placeholder="Last Name" maxlength="15">
                     </div>
-                    <input type="email" id="email" name="email" class="email1" placeholder="Email" required>
-                    <textarea name="message" id="message" class="text" cols="30" rows="10" placeholder="Enter text here." maxlength="500" required></textarea>
+                    <input type="email" id="email" onblur="saveInput();" name="email" class="email1" placeholder="Email" required>
```

Comment:
- [ ] [OPTIONAL] For best practices kindly add all your Javascript in the script (.js) files, here you can target the inputs for exemple with `document.getElementById('email')` and use `addEventListner()` to listen for an event